packages/botonic-plugin-hubtype-analytics/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@botonic/plugin-hubtype-analytics",
+  "version": "0.22.0-alpha.3",
+  "description": "Plugin for tracking in the Hubtype backend to see the results in the Hubtype Dashbord",
+  "main": "./lib/cjs/index.js",
+  "module": "./lib/esm/index.js",
+  "scripts": {
+    "build": "rm -rf lib && ../../node_modules/.bin/tsc -p tsconfig.json && ../../node_modules/.bin/tsc -p tsconfig.esm.json",
```

Comment:
in next iterations it would be good to use the same tsc compiler for everyone but done better. We should reach a point where just installing in the root has same effects as installing them per separate